packages/styles/scss/components/structured-list/_structured-list.scss

```diff
@@ -72,6 +72,10 @@
     }
   }
 
+  .#{$prefix}--with-background-layer {
+    background-color: $layer;
+  }
```

Comment:
Is this necessary since it's already using layer in the `.#{$prefix}--structured-list` selector above in this file?

https://github.com/carbon-design-system/carbon/pull/14726/files#diff-4e80dc6063d7cc3b405f2301c8664f49deaf6b752ccbb5b16f5f4154eebe81f1R35

Reply:
@tay1orjones 
It is not. But the thing is that I wasn't able to redefine the `$layer` in storybook to use as an example, that's why I did this.
If I remove this style and try to use the `.#{$prefix}--structured-list`it will not work in the storybook, since we are not redefine the `$layer`.
From what I remember I didn't find a way to redifine the `$layer`on storybook.

Reply:
I'll remove the storybook that I added. Looking again it doesn't feel like it is worth to have a storybook only for that.